### PR TITLE
Webview plugins written and tested

### LINKED ISSUE #161: Webview checks written and tested

### PR DESCRIPTION
**Pull request by nwalsh1995**
Resolves #161 

Design decisions:

- Added some AST helpers into `helpers.py` which should be useful for other plugins or existing plugins.

Other than that, these plugins are straightforward.

### CODE REVIEW

File: qark/plugins/helpers.py [hunk 1]
```diff
@@ -115,3 +117,40 @@ def java_files_from_files(files):
...
+
+def remove_dict_entry_by_value(dictionary, value):
+    return {k: v for k, v in dictionary.items() if v != dictionary.get(value)}
+
+
+def valid_method_invocation(method_invocation, method_name, num_arguments):
+    """
+    Determines if a `MethodInvocation` has the name `method_name` and the number of arguments `num_arguments`
+
+    :param MethodInvocation method_invocation: the javalang MethodInvocation
```
Comment (neilatli) [inline qark/plugins/helpers.py]: This docstring has a description of the parameter but no type.

File: qark/plugins/webview/helpers.py [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+
+CATEGORY = "webview"
+
+
+def valid_set_method_bool(method_invocation, str_bool, method_name="setAllowFileAccess"):
+    """
+    Determines if the `method_invocation` has the same `method_name` and has an argument of value `str_bool`
+
+    :param MethodInvocation method_invocation:
```
Comment (neilatli) [inline qark/plugins/webview/helpers.py]: This docstring has a type but no description.

File: qark/plugins/webview/javascript_enabled.py [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+from qark.issue import Severity, Issue
+from qark.plugins.helpers import java_files_from_files
+from qark.plugins.webview.helpers import valid_set_method_bool
+from qark.scanner.plugin import BasePlugin
+
+log = logging.getLogger(__name__)
+
+JAVASCRIPT_ENABLED_DESCRIPTION = (
+    "While not a vulnerability by itself, it appears this app has JavaScript enabled in "
+    "this WebView: localWebView If this is not expressly necessary, you should disable "
```
Comment (neilatli) [inline qark/plugins/webview/javascript_enabled.py]: Is `localWebView` meant to be substituted for a variable?

Comment (nwalsh1995) [reply qark/plugins/webview/javascript_enabled.py]: Yeah I think it was broken from the original QARK code. Removed.

**Comment by nocomp** [comment]
hi @nwalsh-lnk @nwalsh1995 in what file is set the url maninthemidle.com cause it gives 404 when tryin to reach for poc from the eploit apk.
thxx a lot

**Pull request merged**